vm/db_test.go

```diff
@@ -51,7 +47,7 @@ func TestPGConnectionPing(t *testing.T) {
 
 	for i, tt := range tests {
 		v := initTestVM()
-		evaluated := v.testEval(t, tt.input, getFilename())
+		evaluated := v.testEvalWithRequire(t, tt.input, getFilename(), "db")
```

Comment:
I'm not quite sure about this change actually. Here's my opinions of it:

1. This deserves another PR, because it's not really related to the subject.
2. I think keep those `require`s in the script is more easy for others to understand how it works. They just need to take a look at the script itself instead of both script and the `v.testEvalWithRequire` function's argument.

BTW the `testEvalWithRequire` only takes one library's name, what if I want to require multiple libraries in my test script? Shouldn't the function be able to require multiple libraries?

Anyway, I personally don't think this is a necessary change.

@saveriomiroddi Can you explain about the reason you want to make this change? Is it because those require statement against the DRY principle?

@goby-lang/maintainers How do you think about this change?

Reply:
> Can you explain about the reason you want to make this change? Is it because those require statement against the DRY principle?

The reason was DRY, correct - for each test file, the `require` is repeated over and over.
It is obviously a requirement for the tests to run, but in a way, from the point of view of each unit test, the `require` statement itself is irrelevant.

> BTW the testEvalWithRequire only takes one library's name, what if I want to require multiple libraries in my test script?

It's a valid point. Since all the UTs always use at most one require, I've implemented a single parameter, but when/if somebody will need multiple requires, variadic args could be used.

> This deserves another PR, because it's not really related to the subject.

Makes sense. I'll remove this commit, and open an issue - then the issue can be discussed and implemented or closed.